config/filter.d/postfix-sasl.conf

```diff
@@ -7,7 +7,7 @@ before = common.conf
 
 [Definition]
 
-_daemon = postfix(-\w+)?/(submission/)?smtp(d|s)
+_daemon = postfix(-\w+)?/((?:submission|smtps)/)?smtpd
```

Comment:
Hi @sebres 

To answer your question: no I am not sure that it will never occur anymore since the daemon name is configurable by the postfix administrator. I can only say that is it ok (and actually a fix) for the default postfix config (which might even get patched by the vendor / distribution / OS).

I'll update the PR with your suggestion :+1: